chroma_core/services/corosync/__init__.py

```diff
@@ -156,7 +152,7 @@ def is_new(peer_node_identifier):
                     #  Raise an Alert - system suppresses duplicates
                     log.debug("Alert notify on %s: active=%s" % (host, not host_reported_online))
                     HostOfflineAlert.notify(host, not host_reported_online)
-                    if host_reported_online == False:
+                    if host_reported_online is False:
```

Comment:
Isn't this change worse? (c.f. [PEP8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations))
```suggestion
                    if not host_reported_online:
```

Reply:
Flake8 seems to be indifferent:

<img width="617" alt="screen shot 2019-01-18 at 9 39 12 am" src="https://user-images.githubusercontent.com/458717/51393144-02f66780-1b05-11e9-903d-8cd63cc6f0d4.png">

I'm also indifferent, so I'll update.

Reply:
Though codefactor follows pep 8 and didn't like `is False`